DirectProgramming/DPC++FPGA/Tutorials/Features/prototype/CSR_and_streaming_interfaces/src/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# To see a Makefile equivalent of this build system:
+# https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC++/ProjectTemplates/makefile-fpga
+
+set(SOURCE_FILE CSR_and_streaming_interfaces.cpp)
+set(TARGET_NAME CSR_and_streaming_interfaces)
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
+
+# FPGA device selection
+if(NOT DEFINED FPGA_BOARD)
+    set(FPGA_BOARD "Arria10")
+    message(STATUS "FPGA_BOARD was not specified.\
+                    \nConfiguring the design to run on the default FPGA device ${FPGA_BOARD} (Intel(R) Arria(R) 10 FPGAs). \
+                    \nPlease refer to the README for information on board selection.")
+else()
+    message(STATUS "Configuring the design to run on FPGA device ${FPGA_BOARD}")
+endif()
+
+# This is a Windows-specific flag that enables exception handling in host code
+if(WIN32)
+    set(WIN_FLAG "/EHsc")
+endif()
+
+# This is for finding the install path to the necessary include files
+if(DEFINED ENV{INTELFPGAOCLSDKROOT})
+   set(SDK_ROOT_PATH $ENV{INTELFPGAOCLSDKROOT})
+else()
+   message(FATAL_ERROR "The INTELFPGAOCLSDKROOT environment variable must be defined for this tutorial to compile.")
+endif()
+
+# A DPC++ ahead-of-time (AoT) compile processes the device code in two stages.
+# 1. The "compile" stage compiles the device code to an intermediate representation (SPIR-V).
+# 2. The "link" stage invokes the compiler's FPGA backend before linking.
+#    For this reason, FPGA backend flags must be passed as link flags in CMake.
+set(EMULATOR_COMPILE_FLAGS "-Wall ${WIN_FLAG} -fintelfpga -DFPGA_EMULATOR -I${SDK_ROOT_PATH}/include/")
+set(EMULATOR_LINK_FLAGS "-fintelfpga")
+set(SIMULATOR_COMPILE_FLAGS "-Wall ${CONSTEXPR_STEPS} ${WIN_FLAG} -fintelfpga -DFPGA_SIMULATOR -I${SDK_ROOT_PATH}/include/")
+set(SIMULATOR_LINK_FLAGS "-fintelfpga -Xssimulation ${SEED_FLAG} -Xstarget=${FPGA_BOARD}")
```

Comment:
SEED_FLAG  is not defined anywhere, if you don't need it, remove it

Reply:
removed